charts/cloudzero-agent/values.yaml

```diff
@@ -59,11 +61,10 @@ validator:
     kubeStateMetrics:
     prometheusNodeExporter:
   # -- Flag to skip validator failure if unable to connect to the CloudZero API.
-  skipValidations: false
   name: env-validator
   image:
-    repository: ghcr.io/cloudzero/cloudzero-charts/cloudzero-agent-validator
-    tag: latest
+    repository: ghcr.io/cloudzero/cloudzero-agent-validator/cloudzero-agent-validator
+    tag: 0.11.0
```

Comment:
just noting for the record because we talked about this a bit earlier, but I am also a fan of pinning to a version instead of using `latest`